crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -1,10 +1,12 @@
 mod render_layers;
 
+use std::collections::HashMap;
```

Comment:
```suggestion
use bevy_utils::HashMap;
```

Reply:
Updated in d266354f282c2943679144f8c0017596fc28e96d.